travel_platform/Html/main.html

```diff
@@ -0,0 +1,111 @@
+{% extends 'blog/base.html' %}
+{% block content %}
+<header>
+  <div class="row head-bkg">
+    <div class="col-md-offset-3 col-md-6 head-text text-center">YOUR<br>TRAVEL</div>
+  </div>
+</header>
+
+<div class="row">
+  <div class="col-md-offset-4 col-md-8"><h2 class="text-center white">Common information</h2>
+  </div>
+
+  <div class="col-md-4 choose">
+    <a href="https://lviv.itea.ua/">
+      <img class="img-responsive" src="pics/planet.jpg" alt="planet">
+      <span class="text-center">Choose a city</span>
+    </a>
+  </div>
+
+  <div class="col-md-8">
+    <div class="col-md-4 choose" id="museums"><a href="https://lviv.itea.ua/">
+      <img class="img-responsive" src="pics/museum.png" alt="museum">
+        <span class="text-center">Museums</span></a>
+    </div>
+
+    <div class="col-md-4 choose" id="attractions"><a href="https://lviv.itea.ua/">
+      <img class="img-responsive" src="pics/attraction.jpg" alt="attraction">
+        <span class="text-center">Attractions</span></a>
+    </div>
+
+    <div class="col-md-4 choose" id="entertainment"><a href="https://lviv.itea.ua/">
+      <img class="img-responsive" src="pics/entertainment.jpg" alt="entertainment">
+        <span class="text-center">Entertainment</span></a>
+    </div>
+
+    <div class="col-md-4 choose" id="hotels"><a href="https://lviv.itea.ua/">
+      <img class="img-responsive" src="pics/otel.jpg" alt="otel">
+        <span class="text-center">Hotels</span></a>
+    </div>
+
+    <div class="col-md-4 choose" id="transport"><a href="https://lviv.itea.ua/">
+      <img class="img-responsive" src="pics/bus.jpg" alt="bus">
+        <span class="text-center">Transport</span></a>
+    </div>
+
+    <div class="col-md-4 choose" id="restaurants"><a href="https://lviv.itea.ua/">
+      <img class="img-responsive" src="pics/restaurant.jpg" alt="restaurant">
+        <span class="text-center">Restaurants</span></a>
+    </div>
+  </div>
+</div>
+
+<div class="row">
+    <div class="col-md-6 text-center"><h2>How to get there?</h2>
+    </div>
+
+    <div class="col-md-4">
+      <form method="get">
+        <h3><input class="form-control text-center" id="location" type="text" value="enter your location"></h3>
+      </form>
+    </div>
+
+    <div class="col-md-2">
+      <button type="submit" class="btn btn-default"><h3>Submit</h3></button>
```

Comment:
Data from form. The action will be assigned when we know how does it interact with the rest of the project. The same situation with another button marked as "Go"